Automattic/jetpack
# WordAds: Updated snippet for DDG expansion

A slightly updated loader to for our project with Duck Duck Go.





#### Changes proposed in this Pull Request:

* A small update to help us differentiate sites for our partnership with Duck Duck Go ads.

#### Jetpack product discussion
pMyNb-4V9-p2

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:






* Enable the Ads module in Jetpack Settings -> Traffic on a Jetpack test site with Premium+ plan.
* Enable some ad units/widgets/etc.
* Ads should load as you'd expect.

#### Proposed changelog entry for your changes:

* No changelog needed.
